README.md

```diff
@@ -14,7 +14,14 @@ To use `haskell-flake` in your Haskell projects, run:
 nix flake init -t github:srid/haskell-flake
 ```
 
-This will generate a template Haskell project with a `flake.nix`. If you already have a Haskell project, copy over this `flake.nix` and adjust accordingly.
+This will generate a template Haskell project with a `flake.nix`. If you already have a Haskell project, run:
+
+
+``` nix
+nix flake init -t github:srid/haskell-flake#flake-only
```

Comment:
One problem here is that this will generate a `flake.nix` hardcoded to use `example.cabal`. But resolving #49 would obviate this issue as we no longer have to specify the package name in the flake.

Reply:
`example` is not a terrible placeholder.

Reply:
- `#default` -> flake only
- `#example` -> full example project